mongoid-observers.gemspec

```diff
@@ -18,7 +18,6 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rails-observers", "~> 0.1.2"
```

Comment:
Ah, that reflects a misunderstanding on my part---I thought adding it to the gem's `Gemfile` would be sufficient. Looks like its maintainers will need to be prodded for a fresh release.